src/components/lead/styles/vars/CdrLead.vars.scss

```diff
@@ -1,10 +1,18 @@
 @mixin cdr-lead-base-mixin() {
   width: 100%;
   line-height: 0;
-  background: linear-gradient(
-    transparent 0 50%,
-    var(--cdr-lead-base-background-color, var(--cdr-color-background-primary, #{$cdr-color-background-primary})) 26% 100%,
-  );
+  &--top {
+    background: linear-gradient(
+      var(--cdr-lead-base-background-color, var(--cdr-color-background-primary, #{$cdr-color-background-primary})) 0 50%,
```

Comment:
is this the surface color - the background color should come from leads parent

Reply:
Meaning the custom prop should be named `--cdr-lead-surface-color`? The tokens for surface don't exist so the fallbacks would still be background color right?